src/main/java/codeu/controller/AjaxGetMessages.java

```diff
@@ -109,11 +111,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
             lastSize++;
         }
 
-        String messagesJson = new Gson().toJson(newMessageList);
-        System.out.println(messagesJson);
+        //Convert each message into an ajaxMessage and itt to ajaxMessageList
+        for(int i = 0; i < newMessageList.size(); ++i){
+          Message actualMessage = newMessageList.get(i);
+          AjaxMessage convertedMessage = AjaxMessage.toAjaxMessage(actualMessage);
+
+          ajaxMessageList.add(convertedMessage);
+        }
```

Comment:
This loop converts all of the new messages to an AjaxMessage right? Why is that done?

Reply:
Oh because originally I was sending a list of Messages as `json`, but the problem was that when I tried to show the author name in the front end I couldn't make it because I couldn't call the datastore from javascript code (that's why in the early versions of this feature it showed a big number instead of the user name), so the only thing that I could use were the attributes from each `Message.java` but the attributes were a lot of `UUIDs` (eg. messageId, userId, etc). So one solution was modifying the `json string` created (retrieving the authorId, querying the datastore for the username, and inserting the username where the authorId was, thus replacing it), but that was going to be a big pain because each message has different content so parsing it was going to be pretty annoying, so I decided to create `AjaxMessage.java` that instead of having the same attributes of `Message.java` it has the attributes that we want to show in the front-end (eg. User that send the message, hour when message was sent, etc), so when I want to display them in the front-end I could just access that information without having to query the datastore. 

Did I made myself clear? Let me know @ppbonds ! 👍